runtime/foucoco/src/currency.rs

```diff
@@ -44,7 +44,7 @@ impl Default for CurrencyId {
 	TypeInfo,
 )]
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
-
+#[repr(u64)]
```

Comment:
Should we not also do this in the `currency.rs` for Amplitude?

Reply:
Agree.Done